LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2020 DevIntent LLC
+Copyright (c) 2021 XLTS.dev https://xlts.dev/
```

Comment:
If I understand the MIT license terms correctly, previous copyright notices must be preserved (but we can add new lines).
Not sure if this applies here. (Also something to consider for our other demos repos.)